packages/nimble-components/src/table-column/number-text/index.ts

```diff
@@ -111,7 +130,12 @@ export class TableColumnNumberText extends TableColumnTextBase {
             case NumberTextFormat.decimal:
                 return new DecimalFormatter(
                     lang.getValueFor(this),
-                    this.decimalDigits ?? defaultDecimalDigits
+                    typeof this.decimalMaximumDigits === 'number'
+                        ? 0
+                        : this.decimalDigits ?? defaultDecimalDigits,
+                    this.decimalMaximumDigits
+                        ?? this.decimalDigits
+                        ?? defaultDecimalDigits
```

Comment:
Is there an issue if `decimalMaximumDigits` is less than `defaultDecimalDigits` (i.e. `decimalMaximumDigits` is 1). It seems that would result in the formatter being given a value for `maximumFractionDigits` that is smaller than `minimumFractionDigits`.

Reply:
No, because if `decimalMaximumDigits` is defined (`typeof this.decimalMaximumDigits === 'number'`), we always pass `0` as the minimum.

Reply:
Oh right...duh. 🤦‍♂️